lib/command.js

```diff
@@ -239,64 +246,73 @@ module.exports = function command (yargs, usage, validation) {
     argv._ = argv._.slice(context.commands.length) // nuke the current commands
     const demanded = commandHandler.demanded.slice(0)
     const optional = commandHandler.optional.slice(0)
+    const parseOptions = {
```

Comment:
perhaps refactor the population of these parser options into a separate helper based on the command string, since they're also required for `.positional()`.